### PR TITLE
[14.0][IMP]account_statement_import_txt_xlsx: additional mapping controls

### PR DESCRIPTION
**Pull request by PicchiSeba**
These changes allow to skip a specified amount of rows/columns during the file parsing phase.
There is also an option that allows to skip completely empy lines.

Please let me know what you think about it and if you like these changes to be made differently.

### CODE REVIEW

File: account_statement_import_txt_xlsx/models/account_statement_import_sheet_mapping.py [hunk 1]
```diff
@@ -61,6 +62,23 @@ class AccountStatementImportSheetMapping(models.Model):
...
     )
+    skip_empty_lines = fields.Boolean(
+        "Skip Empty Lines",
+        default=False,
+        help="Allows to skip empty lines",
+    )
+    offset_column = fields.Integer(
+        "Offset Column",
+        default=0,
+        required=True,
```
Comment (alexey-pelykh) [inline account_statement_import_txt_xlsx/models/account_statement_import_sheet_mapping.py]: Why do you need `required`? Especially if the field is optionally hidden

Comment (PicchiSeba) [reply account_statement_import_txt_xlsx/models/account_statement_import_sheet_mapping.py]: @alexey-pelykh 
Thanks for pointing that out. I forgot about it from an earlier iteration

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]
Please add tests to cover this use-case and I'll proceed with approving

**Comment by PicchiSeba** [comment]
@alexey-pelykh I added a couple of test that cover the new features.
Are they good enough or do we require more in-depth test scenarios?

**Review by alexey-pelykh — APPROVED** [review]

**Comment by alexey-pelykh** [comment]
Please take some time to back- and forward- port the improvement.

**Pull request merged**